tests/test_ra2ce_handler.py

```diff
@@ -2,21 +2,30 @@
 
 import pytest
 
+from ra2ce.analysis.analysis_config_data.analysis_config_data import AnalysisConfigData
+from ra2ce.network.network_config_data.network_config_data import NetworkConfigData
 from ra2ce.ra2ce_handler import Ra2ceHandler
 from tests import test_results
 
 
 class TestRa2ceHandler:
-    def test_initialize_with_no_network_nor_analysis_raises(self):
+    def test_initialize_with_no_network_nor_analysis_does_not_raise(self):
         # 1. Run test.
-        with pytest.raises(ValueError) as exc_err:
-            Ra2ceHandler(None, None)
+        _handler = Ra2ceHandler(None, None)
 
         # 2. Verify final expectations.
-        assert (
-            str(exc_err.value)
-            == "No valid location provided to start logging. Either network or analysis are required."
-        )
+        assert isinstance(_handler, Ra2ceHandler)
+
+    def test_initialize_from_config_does_not_raise(self):
```

Comment:
tests that I miss (potentially with different expected behavior) for the `.from_config` method:

- with `NetworkConfigData` and without `AnalysisConfigData`,
- without `NetworkConfigData` and with `AnalysisConfigData`,
- without `NetworkConfigData` and without `AnalysisConfigData`,

Then, for the "valid" options, I would like to know what happens when you do a `_handler.configure()`, which ones will raise an exception? which ones won't?

Reply:
I parametrized the test, but if I'm not mistaken you had objections to having AnalysisConfigData() as parameter. Or am I mistaken?

Reply:
Configuring a handler requires more data, like graph files? How can I do that easiest without creating an acceptance test?

Reply:
Have you ever included a fixture in a parametrized test as input param?

Reply:
Tests extended